redfish-core/lib/log_services.hpp

```diff
@@ -1085,6 +1108,44 @@ inline void requestRoutesSystemLogServiceCollection(App& app)
                     "/xyz/openbmc_project/object_mapper",
                     "xyz.openbmc_project.ObjectMapper", "GetSubTreePaths", "/",
                     0, std::array<const char*, 1>{postCodeIface});
+
```

Comment:
Could add a comment here on what this is doing. I think this is putting 

"/redfish/v1/Systems/system/LogServices/HardwareIsolation" out if we can find an HardwareIsolation log